webclient/fedora_management/admin.py

```diff
@@ -0,0 +1,73 @@
+# Register your models here.
+import pandas as pd
+from core.ident_cely import get_record_from_ident
+from core.repository_connector import FedoraError, FedoraTransaction
+from django.contrib import admin
+from django.template.response import TemplateResponse
+from django.urls import path
+from django.utils.translation import gettext as _
+from fedora_management.forms import UpdateMetadataFileForm
+from xml_generator.models import ModelWithMetadata
+
+
+class YourCustomAdminSite(admin.AdminSite):
```

Comment:
Prosim prejmenuj podle toho co to ma delat

Reply:
Díky, přejmenováno.